Core/NoOpRunner.Core/Enums/KeyPress.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace NoOpRunner.Core.Enums
+{
+    public enum KeyPress
+    {
+        Up,
+        Right,
+        Left,
+        Space,
+        Down,
+        PowerUp1,
+        PowerUp2
+    }
+}
```

Comment:
Why do you need this?

Reply:
On development branch, this file is part of solution but not there physically, had to readd.

Reply:
Just remove it from the solution instead, I don't think we are using the enum anymore

Reply:
Removed